lib/gtfs_realtime_viz.ex

```diff
@@ -224,8 +224,8 @@ defmodule GTFSRealtimeViz do
 
   @spec trainify_diff([Proto.vehicle_position], [Proto.vehicle_position], Proto.vehicle_position_statuses, String.t, String.t) :: Phoenix.HTML.Safe.t
   defp trainify_diff(vehicles_base, vehicles_diff, status, ascii_train_base, ascii_train_diff) do
-    base = vehicles_with_status(vehicles_base, status) |> Enum.map(& &1.vehicle && &1.vehicle.id)
-    diff = vehicles_with_status(vehicles_diff, status) |> Enum.map(& &1.vehicle && &1.vehicle.id)
+    base = vehicles_with_status(vehicles_base, status) |> Enum.map(& &1.vehicle && &1.vehicle.label)
```

Comment:
the thing i'm not quite sure of is this won't really handle if a train doesn't have a label, or if two trains have the same label

Reply:
why would two trains ahve the same label

Reply:
they might both be blank

Reply:
maybe we could have a function that takes a vehicle and returns the label or the id if the label is blank